nio/events/room_events.py

```diff
@@ -1512,6 +1514,35 @@ def from_dict(cls, parsed_dict):
         )
 
 
+@dataclass
+class ReactionEvent(Event):
+    """Class representing to a m.reaction event.
+
+    Users sometimes wish to respond to a message using emojis. When such
+    responses are grouped visually below the message being reacted to, this
+    provides a (visually) light-weight way for users to react to messages.
```

Comment:
I definitely agree this is helpful context, though it's also not exclusively true since [per the spec](https://spec.matrix.org/v1.7/client-server-api/#mreaction) reactions are not exclusively emoji, or even just groups of emoji; plain text/etc are also perfectly valid.

In the spirit of brevity/technicality, you *could* potentially cut out this context/elaborate on it a bit... but I'm not too bothered either way.

Reply:
Thats why I wrote `reaction/emoji`. But I agree it could be more verbose... If you are not too bothered I would just leave it as it is as I am a bit busy but otherwise I could of course rewrite it.

Reply:
I might just go ahead and make changes directly on the branch then; greatly appreciate the contribution! 🙂